### PR TITLE
find: add Indonesian translation

### PR DESCRIPTION
**Pull request by reinhart1010**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.id/windows/find.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`find {{teks}} {{jalan/menuju/file_atau_direktori}} /v`
+
+- Menghitung jumlah baris dalam file yang mengandung teks tertentu:
+
+`find {{teks}} {{jalan/menuju/file_atau_direktori}} /c`
+
+- Mencari baris-baris dalam file yang mengandung teks tertentu beserta nomor barisnya:
+
+`find {{string}} {{path/to/file_or_directory}} /n`
```
Comment (navarroaxel) [inline pages.id/windows/find.md:20]:
```suggestion
`find {{string}} {{jalan/menuju/file_atau_direktori}} /n`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by bl-ue** [comment]
Hey @reinhart1010! In order to get rid of all of those old commits, and to sync your fork's branches (because we've switched from `master` to `main` as the default branch, etc.), I recommend that you delete your fork and refork it befoure you create another branch for a PR.  Also, don't commit directly to `main` ;)

**Review by bl-ue — APPROVED** [review]
Looks good after @navarroaxel's comments are resolved.

File: pages.id/windows/find.md (hunk 1 above)
Comment (reinhart1010) [reply pages.id/windows/find.md]: Done. https://github.com/tldr-pages/tldr/pull/6066/commits/3b7421979d69e4ed413aac4f54eca9296724dcea

**Review by navarroaxel — APPROVED** [review]

File: pages.id/windows/find.md (hunk 1 above)
Comment (navarroaxel) [reply pages.id/windows/find.md]: Thanks @reinhart1010

**Comment by reinhart1010** [comment]
> Hey @reinhart1010! In order to get rid of all of those old commits, and to sync your fork's branches (because we've switched from `master` to `main` as the default branch, etc.), I recommend that you delete your fork and refork it befoure you create another branch for a PR. Also, don't commit directly to `main` ;)

The `proposal-1` branch was created locally from `master` (through `git branch -m`), so it might be the cause of the problem.

The `proposal-2` and `proposal-3` branches are created from `main` using the same command, and I see that none of those issues occur on https://github.com/tldr-pages/tldr/pull/6067 and https://github.com/tldr-pages/tldr/pull/6068 (except last commits from https://github.com/tldr-pages/tldr/pull/6060 and https://github.com/tldr-pages/tldr/pull/6061 respectively).

**Pull request merged**